### PR TITLE
Adding blog post about using the container image with KRaft mode

### PR DESCRIPTION
**Pull request by gunnarmorling**
@rk3rn3r, @Naros, up for a quick review of this one? Thx!

### CODE REVIEW

File: _posts/2021-08-31-going-zookeeperless-with-debezium-container-image-for-apache-kafka.adoc [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+Since then, the Debezium community has upgraded its https://hub.docker.com/r/debezium/kafka[container image] for Apache Kafka to version 2.8,
+which makes it really simple to start your own explorations around running Kafka without ZooKeeper.
+Note that KRaft mode is an early access feature of Apache Kafka at this point in time and should not be used for production purposes.
+
+== Enabling KRaft Mode
+
+As of Debezium version 1.7, the container image for Kafka supports three new environment variables,
+to be specified when running the image:
+
+* `CLUSTER_ID`: A unique id, such as "5Yr1SIgYQz-b-dgRabWx4g"; if this variable is present, KRaft mode is enabled. Otherwise, the image behaves as before, and must be configured with a reference to ZooKeeper. When using KRaft mode, all nodes of the Kafka cluster must use the same cluster id
```
Comment (rk3rn3r) [inline _posts/2021-08-31-going-zookeeperless-with-debezium-container-image-for-apache-kafka.adoc:30]: maybe node that it needs to be generated with that `kafka-storage.sh` tool? I think this is still needed?

**Comment by rk3rn3r** [comment]
LGTM beside that one minor comment. Does it make sense to show that Kafka Connect is able to connect to that Kafka instance and also maybe that Debezium History Topic/s still work successfully?

File: _posts/2021-08-31-going-zookeeperless-with-debezium-container-image-for-apache-kafka.adoc (hunk 1 above)
Comment (gunnarmorling) [reply _posts/2021-08-31-going-zookeeperless-with-debezium-container-image-for-apache-kafka.adoc:30]: Straight to the pain point, I like that :) I was weaseling around here a bit as I wasn't quite sure whether you _have_ to generate ids using that script or not. I'll explore and clarify.

**Comment by gunnarmorling** [comment]
> Does it make sense to show that Kafka Connect is able to connect to that Kafka instance and also maybe that Debezium History Topic/s still work successfully?

Good point; I'll clarify that ZK removal is a pure implementation detail and fully is transparent to clients like DBZ.

**Review by jpechane — APPROVED** [review]

**Review by Naros — APPROVED** [review]

**Comment by gunnarmorling** [comment]
Reworked to address @rk3rn3r's remarks. Thanks for the quick review all!

**Pull request merged**